Web/starter_project_vanguard/components/layout/index.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import { ReactNode, Suspense } from 'react'
+import Navbar from './navbar'
+
+const RootLayout = ({ children }: { children: ReactNode }) => {
+  return (
+    <div className='px-20 py-8'>
```

Comment:
idk if it is a good idea to add padding at the layout level 
this just means the whole page will have a padding and you won't have the flexibility to decide such things on the page level

eski check what it looks like with the padding because I don't think even the nabber would look fine (not sure about this)

Reply:
so, flexibility is good, but it costs us consistency. I can give padding for the navbar only, and if others give different padding for their pages, they will have different paddings and it looks weird, I thought. but ok I will give the padding for the nav only.